frontends/precinct-scanner/src/screens/poll_worker_screen.tsx

```diff
@@ -509,14 +597,6 @@ export function PollWorkerScreen({
         )}
       </Prose>
       <ScannedBallotCount count={scannedBallotCount} />
-      {isExportingResults && (
-        <ExportResultsModal
-          onClose={() => setIsExportingResults(false)}
-          usbDrive={usbDrive}
-          isTestMode={!isLiveMode}
-          scannedBallotCount={scannedBallotCount}
-        />
-      )}
```

Comment:
@mattroe has been suggesting / asking / arguing to remove the export results option from the poll worker flow for a while, and I decided to do it here. I didn't want to update existing coverage, and we didn't have great coverage on it anyway.

Reply:
I'm not sure I understand the desire of removing it but I guess I defer to @mattroe

Reply:
It's not a feature that we have a reasonable user story for.